chia/wallet/puzzles/utility_macros.clib

```diff
@@ -37,4 +37,12 @@
     )
   )
 
+  (defun gte (atom_a atom_b)
+    (or (> atom_a atom_b) (= atom_a atom_b))
```

Comment:
this is incredibly pedantic but I believe `(not (< atom_a atom_b))` would be more efficient here